contracts-periphery/.env.exmaple

```diff
@@ -0,0 +1,2 @@
+ETH_RPC_URL=
```

Comment:
We could use the addition of README to this subdirectory containing:

- [ ] A quick description of what the subdir is for
- [ ] Instructions to getting setup, including copying the .env.example and what the vars are supposed to be/used for

Reply:
Also is this supposed to be a Rinkeby RPC or mainnet RPC? Or either?

Reply:
It's supposed to be rinkeby since the token address hardcoded in the tests in rinkeby DAI, but agreed docs would be great.

But now that I think of it, we should just write a mock ERC-20 token that way tests will work regardless of which network you fork from. Even more generally, we can just use `deployCode` to deploy Umbra locally and avoid the need to fork at all

Reply:
I liked the no-fork-needed idea, but was having trouble deploying the main Umbra contract. testDeployMainContract() returns deployed address is 0x000...00. https://gist.github.com/garyghayrat/8a2bc2e7d03d3b9493a0b3a520c62ca1.js

Reply:
hey @garyghayrat I think we could improve our description a bit, specifically something like

```markdown
# Umbra Periphery Contracts

Smart contracts that integrate with [Umbra](../README.md), but are not part of the [core protocol contracts](../contracts-core/).
```